### PR TITLE
Add forms

### PR DESCRIPTION
**Pull request by fletcherdodd**
Adding links to contact forms. Where possible specified it was a feature request form so it gave a specific link.

### CODE REVIEW

File: entries/d/deviantart.com.json [hunk 1]
```diff
@@ -3,7 +3,8 @@
     "domain": "deviantart.com",
     "contact": {
       "twitter": "DeviantArt",
-      "facebook": "deviantart"
+      "facebook": "deviantart",
+      "form": "https://www.deviantartsupport.com/en/article/product-feedback-and-feature-requests"
```
Comment (hkamran80) [inline entries/d/deviantart.com.json:7]: Should forms be behind a login screen?

File: entries/g/giphy.com.json [hunk 2]
```diff
@@ -4,7 +4,8 @@
     "contact": {
       "twitter": "GIPHY",
       "facebook": "GIPHY",
-      "email": "support@giphy.com"
+      "email": "support@giphy.com",
+      "form": "https://support.giphy.com/hc/en-us/requests/new?ticket_form_id=360000249791"
```
Comment (hkamran80) [inline entries/g/giphy.com.json]: Per [the schema](https://github.com/2factorauth/twofactorauth/blob/master/tests/schema.json#L103-L118) and the [recently updated `CONTRIBUTING.md`](https://github.com/2factorauth/twofactorauth/blob/master/CONTRIBUTING.md#adding-a-site-that-does-not-support-tfa), only `email` or `form` can be specified, not both. Which would be the best to keep?

File: entries/d/deviantart.com.json (hunk 1 above)
Comment (hkamran80) [inline entries/d/deviantart.com.json:7]: Should forms be behind login screens?

File: entries/g/greatsouthernbank.com.au.json [hunk 3]
```diff
@@ -2,7 +2,8 @@
   "Great Southern Bank": {
     "domain": "greatsouthernbank.com.au",
     "contact": {
-      "facebook": "gsbankau"
+      "facebook": "gsbankau",
+      "form": "https://customervoice.greatsouthernbank.com.au/wix/3/p162973473063.aspx?deployment=1"
```
Comment (hkamran80) [inline entries/g/greatsouthernbank.com.au.json]:
```suggestion
      "form": "https://customervoice.greatsouthernbank.com.au/wix/3/p162973473063.aspx"
```

File: entries/i/imgur.com.json [hunk 4]
```diff
@@ -2,7 +2,10 @@
   "Imgur": {
     "domain": "imgur.com",
     "contact": {
-      "twitter": "imgur"
+      "twitter": "imgur",
+      "facebook": "imgur",
+      "email": "support@imgur.com",
+      "form": "https://help.imgur.com/hc/en-us/requests/new?ticket_form_id=360003147931"
```
Comment (hkamran80) [inline entries/i/imgur.com.json]: Per [the schema](https://github.com/2factorauth/twofactorauth/blob/master/tests/schema.json#L103-L118) and the [recently updated `CONTRIBUTING.md`](https://github.com/2factorauth/twofactorauth/blob/master/CONTRIBUTING.md#adding-a-site-that-does-not-support-tfa), only `email` or `form` can be specified, not both. Which would be the best to keep?

File: entries/l/laybuy.com.json [hunk 5]
```diff
@@ -3,7 +3,8 @@
     "domain": "laybuy.com",
     "contact": {
       "facebook": "PaybyLaybuy",
-      "twitter": "PaybyLaybuy"
+      "twitter": "PaybyLaybuy",
+      "form": "https://help.laybuy.com/hc/en-us/requests/new?ticket_form_id=360004967093"
```
Comment (hkamran80) [inline entries/l/laybuy.com.json:7]:
```suggestion
      "form": "https://help.laybuy.com/hc/en-us/requests/new"
```

File: entries/p/polywork.com.json [hunk 6]
```diff
@@ -3,7 +3,8 @@
 		"domain": "polywork.com",
 		"contact": {
 			"email": "contact@polywork.com",
-			"twitter": "Polywork"
+			"twitter": "Polywork",
+			"form": "https://support.polywork.com/hc/en-us/requests/new?ticket_form_id=4466525327259"
```
Comment (hkamran80) [inline entries/p/polywork.com.json]: Per [the schema](https://github.com/2factorauth/twofactorauth/blob/master/tests/schema.json#L103-L118) and the [recently updated `CONTRIBUTING.md`](https://github.com/2factorauth/twofactorauth/blob/master/CONTRIBUTING.md#adding-a-site-that-does-not-support-tfa), only `email` or `form` can be specified, not both. Which would be the best to keep?

File: entries/s/spotify.com.json [hunk 7]
```diff
@@ -4,7 +4,8 @@
     "url": "https://www.spotify.com/",
     "contact": {
       "facebook": "spotify",
-      "twitter": "SpotifyCares"
+      "twitter": "SpotifyCares",
+      "form": "https://support.spotify.com/contact-spotify-support/"
```
Comment (hkamran80) [inline entries/s/spotify.com.json]: Given that their page is behind a login screen and only provides a link that opens the mail application, I suggest replacing the form URL with the email.

```suggestion
      "email": "support@spotify.com"
```

File: entries/s/stan.com.au.json [hunk 8]
```diff
@@ -3,7 +3,8 @@
     "domain": "stan.com.au",
     "contact": {
       "facebook": "StanAustralia",
-      "twitter": "StanAustralia"
+      "twitter": "StanAustralia",
+      "form": "https://help.stan.com.au/hc/en-us/requests/new?ticket_form_id=944328"
```
Comment (hkamran80) [inline entries/s/stan.com.au.json:7]:
```suggestion
      "form": "https://help.stan.com.au/hc/en-us/requests/new"
```

File: entries/w/westpac.com.au.json [hunk 9]
```diff
@@ -3,7 +3,8 @@
         "domain": "westpac.com.au",
         "contact": {
             "facebook": "Westpac",
-            "twitter": "Westpac"
+            "twitter": "Westpac",
+            "form": "https://banking.westpac.com.au/olfmu/eforms/ConsumerFeedback/#/detailsPage"
```
Comment (hkamran80) [inline entries/w/westpac.com.au.json]:
```suggestion
            "form": "https://banking.westpac.com.au/olfmu/eforms/ConsumerFeedback"
```

**Review by hkamran80 — CHANGES_REQUESTED** [review]

File: entries/d/deviantart.com.json (hunk 1 above)
Comment (fletcherdodd) [reply entries/d/deviantart.com.json:7]: I would argue that in order to contact them (regardless of the method you choose) you need some sort of account, whether that be an email, Twitter or Facebook account. Plus people requesting 2FA I would assume 9 times out of 10 already have an account with the site/service.

File: entries/l/laybuy.com.json (hunk 5 above)
Comment (fletcherdodd) [reply entries/l/laybuy.com.json:7]: I think it's more beneficial to be taken directly to the recommended option to submit the 2FA request/suggestion. Plus depending on if the company's customer service team does reporting or not, having all 2FA requests fall under the same category would be more useful.

File: entries/d/deviantart.com.json (hunk 1 above)
Comment (hkamran80) [reply entries/d/deviantart.com.json:7]: Good argument

File: entries/f/flaticon.com.json [hunk 10]
```diff
@@ -3,7 +3,8 @@
     "domain": "flaticon.com",
     "contact": {
       "twitter": "flaticon",
-      "facebook": "flaticon"
+      "facebook": "flaticon",
```
Comment (hkamran80) [inline entries/f/flaticon.com.json]:
```suggestion
```

The [Facebook test is saying](https://app.circleci.com/pipelines/github/2factorauth/twofactorauth/8433/workflows/0cd3abf9-772f-4937-a62c-ee3ecfbb3289/jobs/8459?invite=true#step-106-4) that it's private or doesn't exist, even though it does.

File: entries/i/imgur.com.json [hunk 11]
```diff
@@ -2,7 +2,9 @@
   "Imgur": {
     "domain": "imgur.com",
     "contact": {
-      "twitter": "imgur"
+      "twitter": "imgur",
+      "facebook": "Imgur",
```
Comment (hkamran80) [inline entries/i/imgur.com.json]:
```suggestion
      "facebook": "100064745678792",
```

As per the [Facebook test](https://app.circleci.com/pipelines/github/2factorauth/twofactorauth/8433/workflows/0cd3abf9-772f-4937-a62c-ee3ecfbb3289/jobs/8459?invite=true#step-106-6)

**Review by hkamran80 — CHANGES_REQUESTED** [review]

File: entries/f/flaticon.com.json [hunk 12]
```diff
@@ -3,7 +3,8 @@
...
+      "form": "https://www.flaticon.com/profile/support"
```
Comment (Carlgo11) [inline entries/f/flaticon.com.json]: Email addresses are preferred over support forms. 
```suggestion
      "email": "support@flaticon.com"
```

**Review by Carlgo11 — DISMISSED** [review]

File: entries/f/flaticon.com.json (hunk 10 above)
Comment (fletcherdodd) [reply entries/f/flaticon.com.json]: It's almost worth getting rid of the Facebook page, the test keeps failing regardless of if you put it's username or the page number

**Review by hkamran80 — APPROVED** [review]

**Pull request merged**